src/brainglobe_napari/tests/test_napari_atlas_representation.py

```diff
@@ -42,3 +55,16 @@ def test_add_to_viewer(make_napari_viewer, expected_atlas_name):
     assert isinstance(reference, Image)
 
     assert allclose(annotation.extent.world, reference.extent.world)
+
+    # check that mesh is slightly smaller, but roughly the same size
```

Comment:
Could this be clarified? Do you mean that the mesh "fits" inside the annotation image (i.e. the whole image, not just the annotated region)?

Reply:
Clarified.